Kernel/Storage/SD/Commands.cpp

```diff
@@ -0,0 +1,194 @@
+/*
+ * Copyright (c) 2023, the SerenityOS developers.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <Kernel/Storage/SD/Commands.h>
+
+namespace Kernel::SD {
+
+constexpr Command build_cmd0()
```

Comment:
itʼ be nice to describe the comments a little (provide a human-readable name?). pointing to the specs would also be a treat! 🙇🏻‍♂️

(i wouldnʼt mind using command index enum names instead of using `cmd✖️` pattern — that could go to the comment instead? 🤔)

Reply:
I can link the "Physical Layer" spec, that one has a big table with all commands, a description of what they do and what they return + their command index. It doesn't tell you how to set all the bits, but you can sort of figure it out from there.

I don't think there's a place in the spec explaining what to set for those bit, but maybe I simply didn't find it. @Ollrogge wrote this file, maybe you have more info?

> (i wouldnʼt mind using command index enum names instead of using cmd✖️ pattern — that could go to the comment instead? thinking)

I don't know, these are only used to fill the array in `get_command`. Having the command number there makes it easier to see where there are gaps

Reply:
No there is no place where the spec explains how to set the bits. I did it the way @mrkct expained it. Looked at what the commands return, how they are used and at other implementations of the SDHC spec and tried to figure it out this way.

Having the command numbers there also makes it easier to find them in the spec because it only talks about e.g. "CMD0" and not about the "GoIdleState command"